cypress/cypress/support/auth-provider-commands/cognito.ts

```diff
@@ -35,8 +35,8 @@ const loginToCognito = (baseUrl: string, username: string, password: string) =>
         () => {
             // Handle cognito portal information
             cy.request(baseUrl + '/env.js').then((resp) => {
-                const OIDC_URL_REGEX = /["']OIDC_URL['"]:\s*['"]([A-Za-z:\-._/0-9]+)['"]/;
-                const OIDC_APP_NAME_REGEX = /["']OIDC_CLIENT_NAME['"]:\s*['"]([A-Za-z:\-._/0-9]+)['"]/;
+                const OIDC_URL_REGEX = /["']*OIDC_URL['"]*:\s*['"]*([A-Za-z:\-._/0-9]+)['"]*/;
+                const OIDC_APP_NAME_REGEX = /["']*OIDC_CLIENT_NAME['"]*:\s*['"]*([A-Za-z:\-._/0-9]+)['"]*/;
```

Comment:
FWIW I think this should probably be `?` instead of `*` right? We wouldn't want to match (and it wouldn't be valid) to have >1 quote. But this works so I'd say not necessary to change it unless you're already making other changes.